### ISSUE TITLE
Update Iosevka to the latest/current version (1.14.0)

### ISSUE THREAD

**Issue by ryanoasis** [ryanoasis/nerd-fonts#229]
#### 🎯 Subject of the issue

Update Iosevka to the latest/current version
Brought up via https://github.com/ryanoasis/nerd-fonts/issues/83#issuecomment-249115721

#### 🔧 Your Setup

N/A

#### ★ Optional

N/A

**Comment by ylluminate** [comment]
This has been updated yet again today with new long needed ligatures.

**Comment by pencilcheck** [comment]
https://github.com/be5invis/Iosevka/issues/56#issuecomment-361358913

Yay!

**Comment by ryanoasis** [comment]
We can repatch for sure but I don't think the ligatures work at all in the Nerd Fonts patched version

**Comment by ryanoasis** [comment]
Re-patched and only did a quick spot check. Feel free to test them out or report issues here. Thanks

**Comment by pencilcheck** [comment]
Cool really appreciate for doing this, I wonder when the new version is going to be officially released?
I submitted a request in homebrew cask font but they said the latest official release is only 1.2.0

**Comment by ryanoasis** [comment]
Thanks @pencilcheck I did see that.

> when the new version is going to be officially released?

TBD. I am actually in my mind targeting end of this month, so there is some release momentum.

> I submitted a request in homebrew cask font but they said the latest official release is only 1.2.0

Right, so I did repatch the Iosevka fonts but now I am holding off on repatching again because of other new additions I would like to get into a release (new glyphs mostly).

I am sorry for the confusion and perhaps I shouldn't have closed the issue via my commit message.

I have added a new label 'pending rebuild...' and better use of 'in progress...' so everyone can get a better idea of progress besides just open/closed issues. I'll apply these labels and re-open this issue now so it better reflects the status 😄
